tests/reduction/reduction_common.h

```diff
@@ -21,7 +21,7 @@ constexpr bool without_property{false};
 
 constexpr size_t number_iterations{10};
 
-constexpr int identity_value{5};
+constexpr int identity_value{0};
```

Comment:
Could this change mask some implementation feature?
Like imagine that internally the identity value is always ignored and set to 0?

Reply:
I changed it to 0 because according to Spec (Table 124 in [4.9.2.1. reduction interface](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#reduction-interface)) "The value of `identity` may be used by the implementation to initialize an unspecified number of temporary accumulation variables." So this value have to be suitable as neutral accumulation value for corresponding operation otherwise result will be unpredictable.
And since only one custom operation without identity (`op_without_identity`) in this tests is basically '+', value is set to zero now.

Reply:
That makes sense.